src/schema/v2/me/collection.ts

```diff
@@ -55,6 +111,9 @@ export const Collection: GraphQLFieldConfig<any, ResolverContext> = {
       private: true,
     })
 
-    return response
+    return {
+      ...response,
+      user_id: meID,
```

Comment:
Nice dependency injection! Altho, why the underscore-ing (if its being injected for use in JS land). Might be a tiny bit nicer to convert from camel -> underscore in the resolver above. #minor

Reply:
Oh, haha thanks for the speedy review. I was actually writing this up just now, see below. (But point taken about the casing)